src/main/java/org/opensearch/flowframework/model/WorkflowNode.java

```diff
@@ -47,7 +47,7 @@ public class WorkflowNode implements ToXContentObject {
     /** The field defining the timeout value for this node */
     public static final String NODE_TIMEOUT_FIELD = "node_timeout";
     /** The default timeout value if the template doesn't override it */
-    public static final String NODE_TIMEOUT_DEFAULT_VALUE = "10s";
+    public static final String NODE_TIMEOUT_DEFAULT_VALUE = "15s";
```

Comment:
Should we have the same timeout for each step or any differences when steps are supposed to take a lot longer (for example local model is an async API that was made async specifically cause it could take 10x longer then remote model)

Reply:
It can be different for workflow steps depending on their response time.  Created https://github.com/opensearch-project/flow-framework/issues/249